renderdoc/android/android.cpp

```diff
@@ -323,6 +323,34 @@ bool CheckAndroidServerVersion(const rdcstr &deviceID, ABI abi)
   return false;
 }
 
+Process::ProcessResult ListPackages(const rdcstr &deviceID, const rdcstr &packageNames)
```

Comment:
This is nitpicky I admit, but `packageNames` is a little misleading as a parameter name when it's also used for non-package-name parameters like `-3`. Just `parameters` would be better I think.